Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/audit/CORE_X12AuditDataTransformConstants.java

```diff
@@ -56,5 +53,6 @@
     public static final String CAQH_X12_CONNECTIVITY_CODED_SYS_NAME = "CAQH CORE Connectivity Metadata";
     public static final String CAQH_X12_CONNECTIVITY_CODED_SYS_DISPLAY_NAME = "Payload Identifier";
     public static final Short PARTICIPANT_OBJ_TYPE_CODE_ROLE_X12 = 20;
```

Comment:
You used the wrapper type here, but the native type above (Short vs. short) - is there a reason for not sticking to one or the other?